PaddleCV/PaddleDetection/tools/infer.py

```diff
@@ -164,6 +164,11 @@ def main():
     clsid2catid, catid2name = get_category_info(anno_file, with_background,
                                                 use_default_label)
 
+    # whether output bbox is normalized in model output layer
+    is_bbox_normalized = False
+    if hasattr(model, 'is_bbox_normalized'):
```

Comment:
check callable? or simply use a member variable instead of method?

Reply:
Done, thanks